src/acceptance/post_upgrade/post_dynamic_policy_test.go

```diff
@@ -39,13 +39,13 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 
 			By("should scale out to 2 instances")
 			helpers.AppSetCpuUsage(cfg, appName, 50, 5)
-			helpers.WaitForNInstancesRunning(appGUID, 2, 10*time.Minute)
+			helpers.WaitForNInstancesRunning(appGUID, 2, 15*time.Minute)
```

Comment:
this should not take this long ... have you checked the scale event is ever sent ?

Reply:
Is the upgrade test actually picking up a brokeness ?

Reply:
how about leaving the upgrade test in a non cleaned up state to debug this or run the test manually with a paused pipline to check ?
how about extra debug activities when it fails like ...  getting logs and polling `cf ash` infinitily till you stop it?

It seriously should not take longer than 5 mins otherwise its broken ... think about this in production to get a new scale  is 15mins acceptable ?